doc/manuals/user/transient_entities.md

```diff
@@ -0,0 +1,124 @@
+# Transient Entities
+
+Orion Context Broker implements *transient entities*. A transient entity is like any other entity 
```

Comment:
A transient Entity is a regular Entity (i.e. it has id/type, a set of attributes, etc.) but with a expiration timestamp. When such point in time is reached the entity is automatically deleted from the Context managed by Orion.

Reply:
Fixe din 78286a073f3b775bed26ebf74bb17da2bea33fe9... expect capitalization, given I Don't Want to Write in This Way :)